### PR TITLE
Update deployment guidelines

### PR DESCRIPTION
**Pull request by Mropat**
## Description
*This PR adds/fixes ...*

### How to prepare for test
- [ ] ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] paxa the environment: `paxa`
- [ ] install on stage (example for hasta):
`bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage.sh [THIS-BRANCH-NAME]`

### How to test
- [ ] do ...

### Expected test outcome
- [ ] check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review
- [ ] code approved by
- [ ] tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Document in ...
- [ ] Deploy this branch
- [ ] Inform to ...

### CODE REVIEW

File: DEPLOYMENT.md [hunk 1]
```diff
@@ -1,20 +1,47 @@
...
+` ### Changed `
+
+` ### Fixed `
+
+6. Review the details once again and merge the branch into master.
+
+
+7. Wait for GitHub actions to process the event, bump version, create release, publish to Dockerhub and PyPi
+
+
```
Comment (henrikstranneheim) [inline DEPLOYMENT.md:31]: Should not this also be added:
1. Log into relevant serves with `ssh <hasta/clinical-db>`.

File: DEPLOYMENT.md [hunk 2]
```diff
@@ -1,20 +1,47 @@
...
-    1. `bash update-cg-stage.sh master`
-    1. make sure that installation was successful
-1. Deploy master to production
-    1.`up`
-    1. `bash update-cg-prod.sh`
-    1. make sure that installation was successful
-1. Take a screenshot and post as a comment on the PR. Screenshot should include environment and that it succeeded
+When all tests done and successful and PR is approved by codeowners, follow these steps:
+
+1. Select "Squash and merge" to merge branch into default branch (master/main)
```
Comment (henrikstranneheim) [inline DEPLOYMENT.md:5]:
```suggestion
1. Select "Squash and merge" to merge branch into default branch (master/main).
```

File: DEPLOYMENT.md [hunk 3]
```diff
@@ -1,20 +1,47 @@
...
+
+
+2. A prompt for writing merge commit message will pop up
```
Comment (henrikstranneheim) [inline DEPLOYMENT.md:8]:
```suggestion
2. A prompt for writing merge commit message will pop up.
```
Just to be consistent

File: DEPLOYMENT.md [hunk 4]
```diff
@@ -1,20 +1,47 @@
...
+` ### Added `
+
+` ### Changed `
+
+` ### Fixed `
+
+6. Review the details once again and merge the branch into master.
+
+
+7. Wait for GitHub actions to process the event, bump version, create release, publish to Dockerhub and PyPi
```
Comment (henrikstranneheim) [inline DEPLOYMENT.md:29]:
```suggestion
7. Wait for GitHub actions to process the event, bump version, create release, publish to Dockerhub and PyPi.
```

File: DEPLOYMENT.md [hunk 5]
```diff
@@ -1,20 +1,47 @@
...
+8. Deploy master to stage:
+
+    1. `us`
+    2. Request stage environment `paxa` and follow instructions
+    3. `bash update-cg-stage.sh master`
+    4. make sure that installation was successful
+   
+
+10. Deploy master to production
+     1. `up`
```
Comment (henrikstranneheim) [inline DEPLOYMENT.md:41]:
```suggestion
9. Deploy master to production

          1. `up`
```

File: DEPLOYMENT.md [hunk 6]
```diff
@@ -1,20 +1,47 @@
...
+     2. `bash update-cg-prod.sh`
+     3. make sure that installation was successful
+
+
+11. Take a screenshot or copy log text and post as a comment on the PR. Screenshot should include environment and that it succeeded
```
Comment (henrikstranneheim) [inline DEPLOYMENT.md:46]:
```suggestion
11. Take a screenshot or copy log text and post as a comment on the PR. Screenshot should include environment and that it succeeded.
```

File: DEPLOYMENT.md [hunk 7]
```diff
@@ -1,20 +1,47 @@
...
+
+
+3. Find the title of the pull request already pre-filled in the merge commit title, or copy and paste 
+the title if not.
+
+
+4. Append version increment value `( major | minor | patch )` to specify what kind of release is to be created.
+
+
+5. Fill in mkdown formatted changelog in merge commit comment details:
```
Comment (henrikstranneheim) [inline DEPLOYMENT.md:18]:
```suggestion
5. Fill in markdown formatted changelog in merge commit comment details:
```

File: DEPLOYMENT.md [hunk 8]
```diff
@@ -1,20 +1,47 @@
...
+
+7. Wait for GitHub actions to process the event, bump version, create release, publish to Dockerhub and PyPi
+
+
+8. Deploy master to stage:
+
+    1. `us`
+    2. Request stage environment `paxa` and follow instructions
+    3. `bash update-cg-stage.sh master`
+    4. make sure that installation was successful
```
Comment (henrikstranneheim) [inline DEPLOYMENT.md:37]:
```suggestion
    4. Make sure that installation was successful
```

File: DEPLOYMENT.md [hunk 9]
```diff
@@ -1,20 +1,47 @@
...
+   
+
+10. Deploy master to production
+     1. `up`
+     2. `bash update-cg-prod.sh`
+     3. make sure that installation was successful
```
Comment (henrikstranneheim) [inline DEPLOYMENT.md:43]:
```suggestion
     3. Make sure that installation was successful
```

**Review by henrikstranneheim — APPROVED** [review]
Great! Good that you also removed some old stuff

File: DEPLOYMENT.md (hunk 1 above)
Comment (Mropat) [reply DEPLOYMENT.md:31]: done!

**Pull request merged**